### PR TITLE
Add Portuguese translation (light version)

### PR DESCRIPTION
**Pull request by gjvnq**
This is a trimed down version of pull request #109.

I still need to double check the translation to find any mistakes, mispellings, that sort of thing. But I believe it's already possible to review the code changes.

There are two changes not really necessary for the translation itself but that I felt should be included:
1. The changes on `_global.scss` that make the fragment links work better.
2. The partial replacement of the Gothic A1 font with the Inter font as the previous lacks just a few characters necessary in Portuguese, namely á, í, ó, ú, ã, õ, and their upper case versions.

Notes about the translation:
1. I added info specifically about the situation in Brazil in _Diagnosticando Disforia de Gênero_ (Diagnosing Gender Dysphoria) and _Tratando Disforia de Gênero_ (Treating Gender Dysphoria).
  1. I commented on how the Brazilian Federal Medicine Council (CFM) changed its definition of transsexualism [sic] over time.
  3. I summarized the current rules of CFM for gender affirming care.
2. I inserted myself on _Intro. Segunda Puberdade Estrogênica_ (Estrogenic Second Puberty 101). Where it said “including by yours truly” it became “incluindo a autora e a tradutora desse documento” (including the author-♀ and the translator-♀ of this document). (the -♀ notation means that the original word was of the feminine gramatical gender)
4. In some places I replaced the _-sexual_ and _-philic_ suffixes with _-afetiv(o/a/e)_ which encompass both sexual and romantic love. E.g. _gynosexual_ became _ginoafetiva_ instead of _ginofílica_. E.g. _monosexual_ became _monoafetiva_.
5. I haven't translated all the images but I think that this doesn't compromise the understanding of the text and I plan to fix this in a future PR.

The attached file [gdb---pt-light.wacz.txt](https://github.com/GenderDysphoria/GenderDysphoria.fyi/files/9752968/gdb---pt-light.wacz.txt) (remember to rename `.wacz.txt` to `.wacz` before trying to view it on https://replayweb.page) is a preview of what the website will look like after this PR.

### CODE REVIEW

File: build/lib/slugify.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const slugs = require('slugify');
 
 module.exports = exports = function slugify (s) {
-  const result = slugs(s, { remove: /[*+~.,()'"!?:@/\\]/g }).toLowerCase();
+  const result = s.trim().replace(/[^\p{L}\p{N}]+/ug, '-').replace(/^-+/, '').replace(/-+$/, '').toLowerCase();
```
Comment (Twipped) [inline build/lib/slugify.js:4]: Have you checked if this changes any of the existing urls? I don't want to break any links.

**Comment by gjvnq** [comment]
In my experience it didn't break anything but I'll write a short testing
program just to be safe.

I had made this change on slugify.js because the links to section headers
were not working properly.

**Comment by Twipped** [comment]
I did a diff of the build from master vs this branch, and it's looking like they're all identical. We're good to merge, great work!

**Comment by gjvnq** [comment]
> We're good to merge, great work!

Thanks! I'm flattered!

**Pull request merged**